src/components/Turn/Turn.jsx

```diff
@@ -3,6 +3,8 @@ import socket from '../../socket';
 import axios from 'axios';
 import AllCards from '../Cards/AllCards'
 import Message from '../Message/Message'
+import PlayerCards from '../Cards/PlayerCards';
+import style from './Turn.module.css';
```

Comment:
Some files use `styles` and some use `style`. You might want to keep this consistent.